nidirect_common/inc/landing_page.redirects.inc

```diff
@@ -0,0 +1,129 @@
+<?php
+
+/**
+ * @file
+ * Contains functions Layout Builder search indexing.
```

Comment:
Think you need to update the file description

Reply:
*facepalm* My copy/pasta secrets must not be known :)

Good spot!